staging/src/k8s.io/apiserver/pkg/cel/template/template.go

```diff
@@ -0,0 +1,191 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package template
```

Comment:
This package doesn't need to be under `k8s.io/apiserver/pkg/cel`. I can also put it under `k8s.io/dynamic-resource-allocation`. But as it stands now, it is generic and not specific to DRA, so perhaps `apiserver` is better?